tests/conftest.py

```diff
@@ -52,6 +53,26 @@ def eager_task_loop_factory() -> asyncio.AbstractEventLoop:
     )
 
 
+@pytest.fixture(autouse=True)
+def blockbuster() -> Iterator[BlockBuster]:
+    with blockbuster_ctx(
+        "anyio", excluded_modules=["anyio.pytest_plugin", "anyio._backends._asyncio"]
+    ) as bb:
+        bb.functions["os.read"].can_block_in(
+            "asyncio/base_events.py", "subprocess_exec"
```

Comment:
I'll upstream this one to blockbuster. `subprocess_shell` is already excluded for the same reason.